ts/features/pn/utils/dates.ts

```diff
@@ -0,0 +1,122 @@
+import * as t from "io-ts";
+import { PatternString } from "italia-ts-commons/lib/strings";
+
+const isDate = (v: t.mixed): v is Date => v instanceof Date;
+
+/**
+ * Accepts short formats (ie. "2018-10-13")
+ * with or without time and timezone parts.
+ */
+export const DateFromString = new t.Type<Date, string>(
```

Comment:
Why we are not using import { DateFromString } from "@pagopa/ts-commons/lib/dates";
?